code/modules/mob/mob.dm

```diff
@@ -1078,19 +1078,50 @@
 		if(E.mouse_pointer)
 			client.mouse_pointer_icon = E.mouse_pointer
 
+/**
+ * Can this mob see in the dark
+ *
+ * This checks all traits, glasses, and robotic eyeball implants to see if the mob can see in the dark
+ * this does NOT check if the mob is missing it's eyeballs. Also see_in_dark is a BYOND mob var (that defaults to 2)
+**/
+/mob/proc/has_nightvision()
+	return see_in_dark >= NIGHTVISION_RANGE
 
-///This mob is abile to read books
+/// Can this nerd read
 /mob/proc/is_literate()
 	return FALSE
 
+/**
+ * Checks if there is enough light where the mob is located
+ *
+ * Args:
+ *  light_amount (optional) - A decimal amount between 1.0 through 0.0 (default is 0.2)
+**/
+/mob/proc/has_light_on_turf(light_amount = LIGHTING_TILE_IS_DARK)
+	var/turf/mob_location = get_turf(src)
+	var/area/mob_area = get_area(src)
+
+	if(mob_location.get_lumcount() > light_amount)
+		return TRUE
+	else if(!mob_area.static_lighting)
+		return TRUE
```

Comment:
lmao is static_lighting the opposite of what it's supposed to be?
Would `IS_DYNAMIC_LIGHTING` work better here?

Reply:
> lmao is static_lighting the opposite of what it's supposed to be?
> Would `IS_DYNAMIC_LIGHTING` work better here?

are you a goober kat? its literally in the pr you linked you silly coder

![image](https://user-images.githubusercontent.com/62388554/230822871-a142b144-87e1-469a-8899-c1106533f860.png)